website/docs/patterns/form-patterns/partials/validation/validation.md

```diff
@@ -59,6 +59,18 @@ Validating a form on submission can occur on the client or the server and can be
   <source src="/assets/patterns/form-patterns/validation-on-submit-example.mp4" />
 </video>
 
+### Delayed validation
+
+Delayed validation occurs on the client and refers to validating the field after a lapse in keystrokes and a specific interval of time, e.g., `500ms` or 0.5 seconds. Once the user has stopped input into the field and after the interval of time has expired, the field is validated without an `onblur` event occurring.
+
+We recommend **against** using this method because it is invasive and may result in unintended validation errors by assuming a user has completed filling out a field.
```

Comment:
Before recommending so strongly against it, maybe we could ask the ambassadors what is their current method(s) of validation in their products (I suspect this could be the most common? expecially in Cloud UI)?

Reply:
Yes, that's a good idea. I think the reasoning for recommending against this method is pretty valid, it can be a bit of a rough user experience, but I'll see what our consumers say.

Reply:
Yeah, I see what you mean. OK then.